semantic_release/dist.py

```diff
@@ -7,7 +7,7 @@
 
 def build_dists():
     commands = config.get('semantic_release', 'build_commands')
```

Comment:
Can we change `build_commands` to `build_command` since we are doing a breaking change already?

```suggestion
    commands = config.get('semantic_release', 'build_command')
```